test/libsolidity/syntaxTests/modifiers/modifier_call_unimplemented_2.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Why not GPL?

Reply:
This should have not been in there, I only added that so `./solc/solc` during debugging tests did not complain. will remove.